src/index.html

```diff
@@ -7,6 +7,305 @@
+          <div class="card__reviews-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+          
+          <div class="card__price">
+            <p class="card__price-heading">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+          <a 
+            href="#"
+            class="card__buy-link"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+        <div class="wrapper__card card">
+          <img 
+            src="./images/imac.jpeg"
+            alt="image"
+            class="card__image"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__reviews-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+          
+          <div class="card__price">
+            <p class="card__price-heading">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+          <a 
+            href="#"
+            class="card__buy-link"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+        <div class="wrapper__card card">
+          <img 
+            src="./images/imac.jpeg"
+            alt="image"
+            class="card__image"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__reviews-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__reviews">Reviews: 5</div>
+          </div>
+          
+          <div class="card__price">
+            <p class="card__price-heading">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+          <a 
+            href="#"
+            class="card__buy-link"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </div>
+      </div>
+    </main>
   </body>
-</html>
+</html>
```

Comment:
add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)